.github/workflows/ci-backend.yml

```diff
@@ -67,32 +67,16 @@ jobs:
             fail-fast: false
             matrix:
                 python-version: ['3.7.8', '3.8.5', '3.9.0']
-        services:
-            postgres:
-                image: postgres:12
-                env:
-                    POSTGRES_USER: postgres
-                    POSTGRES_PASSWORD: postgres
-                    POSTGRES_DB: postgres
-                ports: ['5432:5432']
-                options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
-            clickhouse-server:
-                image: yandex/clickhouse-server
-                ports:
-                    - '8123:8123'
-                    - '9000:9000'
-                    - '9440:9440'
-                    - '9009:9009'
         steps:
             - uses: actions/checkout@v1
               with:
                   fetch-depth: 1
-
+            - name: Start stack with Docker Compose
+              run: docker-compose -f ee/docker-compose.ch.yml up -d db clickhouse
```

Comment:
Q: Is this susceptible to race conditions? Given the command exits before the service is actually set up.

The service description above did seem to at least wait for pg to come up.

Reply:
I haven't seen it be an issue yet and this is the suggested way of doing this. We could always add another step that waits for PG, CH, and Kafka if it does become a problem. I'd have that be a separate PR though.